public/importHeaderData.js

```diff
@@ -0,0 +1,1028 @@
+const headerData = {
```

Comment:
will you add a comment to this file also explaining how it interacts with the static pages entry script. I am mostly thinking for the part where we co-exist in production with content-build, we will likely use that generated data file, but we'll still need the lines in here to add it to the window object

Reply:
Added a comment with code to clarify